open-metadata-resources/open-metadata-deployment/docker/configure/pom.xml

```diff
@@ -32,17 +32,45 @@
                 <artifactId>dockerfile-maven-plugin</artifactId>
```

Comment:
I think this should work - the two step process is needed to get the latest approach working.
I can't test in the next few hours as the full build is too long, so will raise a new issue if any issues observed

Reply:
I tested on my side and it worked, but would indeed be good to have a peer-reviewed test =)